orb/orb.yml

```diff
@@ -72,6 +77,7 @@ jobs:
                 /harpocrates \
                   --format json \
                   --output "<< parameters.output >>" \
+                  --append=<< parameters.append >> \
```

Comment:
I dont really like this part, but cant get it to work without the `"`

Reply:
It's properly just because of some bash thing, with the `true` - we do they same thing with the `--validate` option